.circleci/config.yml

```diff
@@ -40,6 +40,9 @@ jobs:
             helm repo add pangeo https://pangeo-data.github.io/helm-chart/
             helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
             helm repo add dask-gateway https://dask.org/dask-gateway-helm-repo/
+            helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+            helm repo add grafana https://grafana.github.io/helm-charts
+            helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```

Comment:
This should be replaced with https://charts.helm.sh/stable, the non deprecated location